File: python/src/Behavioral/Visitor/GameInfo.py
#//GameInfo.java - three of three concrete Visitees
from AbstractTitleInfo import AbstractTitleInfo

class GameInfo(AbstractTitleInfo):
	def __init__(self,titleName) :
		super(GameInfo,self).__init__(titleName)

	def accept(self,titleBlurbVisitor) :
		 titleBlurbVisitor.visit(self)

File: scala/src/main/scala/Behavioral/Visitor/GameInfo.scala
package Behavioral.Visitor;


//GameInfo - three of three concrete Visitees

public class GameInfo extends AbstractTitleInfo {  
   public GameInfo(String titleName) {
       this.setTitleName(titleName);
   }    
   
   public void accept(TitleBlurbVisitor titleBlurbVisitor) {
       titleBlurbVisitor.visit(this);
   }
}